src/app/typescript/v5/react/wallets/page.mdx

```diff
@@ -0,0 +1,92 @@
+import { Breadcrumb, createMetadata } from "@doc";
```

Comment:
For example: This page has been deleted and replaced with a different page - and this PR adds it back

Reply:
deleted